src/components/sidebar/Sidebar.jsx

```diff
@@ -144,6 +144,71 @@ export function Sidebar() {
               </label>
             </div>
           </li>
+          <li class="sidebar-list-item">
```

Comment:
change class to className in whole project.

Reply:
Elsewhere its fine...this i forgot ..will change..thanks